pulsar-broker-common/src/main/java/org/apache/pulsar/broker/authorization/PulsarAuthorizationProvider.java

```diff
@@ -451,42 +448,31 @@ public CompletableFuture<Boolean> allowNamespaceOperationAsync(NamespaceName nam
             log.debug("Check allowNamespaceOperationAsync [{}] on [{}].", operation.name(), namespaceName);
         }
 
-        return validateTenantAdminAccess(namespaceName.getTenant(), role, authData)
```

Comment:
I think this call rightly belongs in the `AuthorizationProvider` impelmentation.